src/Apps/Artist/ArtistApp.tsx

```diff
@@ -6,12 +6,12 @@ import { HorizontalPadding } from "Apps/Components/HorizontalPadding"
 import { track } from "Artsy/Analytics"
 import * as Schema from "Artsy/Analytics/Schema"
 import { PreloadLinkState } from "Artsy/Router/Components/PreloadLink"
+import { LoadingArea } from "Components/v2/LoadingArea"
```

Comment:
What does `Components/v2` mean?

Reply:
cc @damassi this is our convention, all new components go there (vs old `Styleguide`). I'm just following the convention! Chris will be able to explain reasoning more.